language/diem-framework/modules/DiemConsensusConfig.move

```diff
@@ -0,0 +1,78 @@
+address 0x1 {
```

Comment:
Hello I am here to make useless nits for the styling of this move! So sorry in advance :) 

I really want an autoformatter some day...

The content of the module looks fine! Even though I don't quite understand what is for (but that's not my domain)

Reply:
it's just a storage of arbitrary serialized bytes that consensus use to coordinate, imagine things like consensus_protocol_version: u64;

Reply:
Are the fields well-understood enough to use a struct instead of arbitrary bytes? We've stored configs as raw bytes before, then ended up switching to a struct to simplify writing update logic. E.g., if the config has 10 fields and you only want to update `consensus_protocol_version`, it can be a pain to fetch the values of the old config from on-chain, swap out the one you want to change, and then push back the entire blob.

Reply:
I have no idea how the config would look like and would like an easier update process. my initial though is to have 
```
enum ConsensuConfig {
  V1(..),
  V2(..),
}
```
to provide flexibility for us to add/remove any fields, hopefully the overall size would be small enough (a few numbers I guess).

Reply:
Ah. In that case, I agree that bytes make sense.